forge-gui/res/cardsfolder/upcoming/kardum_patron_of_flames.txt

```diff
@@ -0,0 +1,19 @@
+Name:Kardum, Patron of Flames
+ManaCost:2 R R
+Types:Legendary Creature Demon
+PT:4/3
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, put a flame counter on it, then seek a card with mana value equal to the number of flame counters on it and exile that card face down.
+SVar:TrigCounter:DB$ PutCounter | CounterType$ FLAME | CounterNum$ 1 | ValidCards$ Self | SubAbility$ DBSeek
+SVar:DBSeek:DB$ ChangeZone | Origin$ Library | Destination$ Exile | ExileFaceDown$ True | AtRandom$ True | NoShuffle$ True | NoLooking$ True | NoReveal$ True | ChangeNum$ 1 | ChangeType$ Card.cmcEQX+YouCtrl | RememberChanged$ True 
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigReturn | TriggerDescription$ When NICKNAME dies, put the cards exiled with it into their owner's hand.
```

Comment:
Add `At the beginning of the end step of your next turn, discard those cards.` to this TriggerDescription, too

Reply:
Always double check to be sure the card detail pane matches the Oracle
![image](https://user-images.githubusercontent.com/103371817/185661974-d7e24c4f-be23-4158-a9c0-ba754d4e6ba3.png)